packages/react-baseline-inputs/src/types.ts

```diff
@@ -180,9 +185,8 @@ export type MaskedInputProps = FieldInputProps &
 
 export type RadioGroupProps = FieldInputProps &
   HTMLProps<HTMLInputElement> &
-  ValueProps<string | null> & {
-    options?: Array<OptionProps | string>;
-  };
+  ValueProps<string | null> &
+  ToggleGroupProps;
```

Comment:
Ideally, the only changes that would be necessary in this file would be adding `& FieldSetProps` to `CheckboxListProps` and `RadioGroupProps`